### PR TITLE
Add configuration files for tomorrow's test.

### PR DESCRIPTION
**Pull request by mpereira**
This depends on https://github.com/mesosphere/spark-build/pull/394

### CODE REVIEW

File: scale-tests/configs/2018-08-22-mvdrf112s.env [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+# Batch GPU jobs configuration #################################################
+
+GPU_DISPATCHERS_JSON_OUTPUT_FILE_URL="https://${TEST_S3_BUCKET}.s3.amazonaws.com/${TEST_S3_FOLDER}/${GPU_DISPATCHERS_JSON_OUTPUT_FILE}"
+
+GPU_APP_ID="/${SERVICE_NAMES_PREFIX}gpu-batch-workload"
+GPU_SCRIPT_CPUS=2
+GPU_SCRIPT_MEM=4096
+GPU_DOCKER_IMAGE='samvantran/spark-dcos-gpu:metrics'
+GPU_SUBMITS_PER_MIN=1
```
Comment (samvantran) [inline scale-tests/configs/2018-08-22-mvdrf112s.env]: can you change this to 5? This was accidentally set to 1 last time and resulted in a very low amount of gpu jobs. Setting to 5 should result in all gpu agents working majority of the time.

File: scale-tests/configs/2018-08-22-testdrf112s.env [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Depends on:
+# - TEST_NAME
+# - TEST_S3_BUCKET
+# - TEST_S3_FOLDER
+
+# Cluster configuration ########################################################
+
+CLUSTER_URL="https://testdrf112s.scaletesting.mesosphe.re"
```
Comment (samvantran) [inline scale-tests/configs/2018-08-22-testdrf112s.env]: did this work on the test cluster? I think I remember reading that the test cluster only had 3 masters, 3 agents? Last time the cluster was so small it was not possible to verify the entire small-scale script could run to completion.

Comment (mpereira) [reply scale-tests/configs/2018-08-22-testdrf112s.env]: It did not. The cluster had 10 agents, but they only had 4 cpus each, so I was only able to install ZK+Kafka+Cassandra, no space left for Spark dispatchers.

File: scale-tests/configs/2018-08-22-mvdrf112s.env (hunk 1 above)
Comment (mpereira) [reply scale-tests/configs/2018-08-22-mvdrf112s.env]: Bumped to 5: https://github.com/mesosphere/spark-build/pull/395/commits/7fa53dccdb3d113cee431097876898d20700d537

**Pull request merged**